src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -141,6 +142,7 @@ public JsonSerializerOptions(JsonSerializerOptions options)
             _ignoreReadOnlyProperties = options._ignoreReadOnlyProperties;
             _ignoreReadonlyFields = options._ignoreReadonlyFields;
             _includeFields = options._includeFields;
+            _newLine = options._newLine;
```

Comment:
The new field must also be taken into account in the `JsonSerializerOptions.Caching.cs` methods.